fix: compare `effective balances` as big numbers

enzo-bitfly opened the pull request:
See: BEDS-1258
See: BEDS-1271

marcel-bitfly commented on frontend/utils/bigMath.ts:
```diff
@@ -24,3 +24,17 @@ export const divideBigNumbers = (
   return `${quotient}`
 }
+
+/**
+ * Returns:
+ *   - `-1` if `leftHandValue` is smaller than `rightHandValue`
+ *   - `0` if both values are equal
+ *   - `1` if `leftHandValue` is greater than `rightHandValue`
+ */
+export const compareBigNumbers = (
+  leftHandValue: number | string,
+  rightHandValue: number | string,
+) => {
+  assertIsNumber(leftHandValue, rightHandValue)
+  return Big(leftHandValue).cmp(Big(rightHandValue))
+}
```
**suggestion (blocking):** plz use `.gte method`

I think otherwise you are missing the `0` case and working with `number abstraction` does not work that well

See: https://mikemcl.github.io/big.js/#gte

enzo-bitfly replied:
Done

marcel-bitfly commented:
**thought**(non-blocking): I don't know if it would be helpful to mention where `effective balance` is `compared` in the `commit message` 🤷

enzo-bitfly committed:
fix: compare `effective balances` as big numbers in `dashboard` page and `DashboardValidatorManagementModal.vue`

See: BEDS-1258
See: BEDS-1271

marcel-bitfly commented on frontend/components/dashboard/DashboardValidatorManagementModal.vue:
```diff
@@ -307,7 +307,7 @@ const hasReachedLimit = computed(() => {
     return false
   }
-  return currentEffectiveBalance.value >= effectiveBalanceLimitPerDashboard.value
+  return isGreaterOrEqual(currentEffectiveBalance.value, effectiveBalanceLimitPerDashboard.value)
```
**suggestion (non-blocking):** I think it reads easier (and a bit fewer to think about 🤷 )
```suggestion
  return isGreaterEquals(currentEffectiveBalance.value, effectiveBalanceLimitPerDashboard.value)
```

enzo-bitfly replied:
Done

enzo-bitfly committed:
refactor: remove `BigNumbers` suffix from `bigMath` methods

- the difference between these methods and `arithmetic operators` is evident without the need of a suffix
